src/ReactiveUI/CommandBinding.cs

```diff
@@ -9,6 +9,9 @@
 
 namespace ReactiveUI
 {
+    /// <summary>
+    /// Various helpers to bind View controls and ViewModel commands together
+    /// </summary>
     public static class CommandBinder
     {
         static ICommandBinderImplementation binderImplementation;
```

Comment:
There were more major things I felt like doing, but stopped myself...

This interface is internal, and only used by the concrete implementation, I'm not sure what benefit this split into a fair number of chunky classes that only call each other and all coexist in the one file is. I'd be tempted to combine into the one static class at this level and that should hopefully reduce the clutter/make it easier to follow.